cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractConnectOperator.java

```diff
@@ -641,15 +664,15 @@ protected Future<ReconcileResult<ClusterRoleBinding>> connectInitClusterRoleBind
     // Static utility methods and classes
 
     private static Function<Map<String, Object>, Future<ConnectorStatusAndConditions>> createConnectorStatusAndConditions() {
-        return statusResult -> Future.succeededFuture(new ConnectorStatusAndConditions(statusResult, List.of(), List.of(), null));
+        return statusResult -> io.vertx.core.Future.succeededFuture(new ConnectorStatusAndConditions(statusResult, List.of(), List.of(), null));
```

Comment:
why we needed the full package? (same for the next lines).

Reply:
Strange, looks like IntelliJ decided to add them and I did not notice. Thanks for catching it.